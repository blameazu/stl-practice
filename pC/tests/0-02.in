10
3 221066297
3 125195186
1 129680597
1 183820348
2 398810904
3 536358721
1 432387214
2 60576614
1 140442775
3 592180838
