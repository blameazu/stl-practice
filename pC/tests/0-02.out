Grandma ta las!
Grandma ta las!
Grandma ta las!
Grandma ta las!
